libs/core/src/components/pds-checkbox/pds-checkbox.scss

```diff
@@ -1,21 +1,21 @@
 :host {
-  --background-color: var(--pine-color-base-white);
-  --background-color-disabled: var(--pine-color-neutral-grey-200);
-  --background-color-hover: var(--pine-color-neutral-grey-100);
-  --border: var(--pine-border-interactive);
-  --border-color-disabled: var(--pine-color-neutral-charcoal-100);
-  --border-color-hover: var(--pine-border-interactive-hover-color);
-  --border-color-icon: var(--pine-color-base-white);
-  --border-radius: var(--pine-border-radius-xs);
+  --border: var(--pine-border-width-thin) solid var(--pine-color-grey-400);
+  --border-color-disabled: var(--pine-color-charcoal-100);
+  --border-color-icon: var(--pine-color-white);
+  --border-color-interactive-hover: var(--pine-border-width-thin) solid var(--pine-color-grey-500);
+  --border-radius: var(--pine-border-radius-050);
   --box-shadow-focus: 0 0 0 2px var(--pine-color-primary-200);
   --box-shadow-focus-invalid: 0 0 0 2px var(--pine-color-red-200);
-  --color-checked: var(--pine-color-neutral-charcoal-500);
-  --color-disabled: var(--pine-color-neutral-grey-300);
+  --color-background: var(--pine-color-white);
```

Comment:
Here as well

Reply:
Not sure I understand the ask here.

Reply:
Whoops sorry looks like my other comment didn't go through. I was going to ask to add context to where/how the color would be used, but it looks like you've updated that. All good.